docs/Quick Reference.md

```diff
@@ -38,7 +38,7 @@ This table summarizes the filters and other options available inside a `tasks` b
 |                                                                                                                                                                                                                                                                                                 |                                             | `group by backlink`    | `hide backlink`        |                                              |
 | **[[Filters#Description\|Description]]**, **[[Filters#Tags\|Tags]]** and other odds and ends                                                                                                                                                                                                    |                                             |                        |                        |                                              |
 | `description (includes, does not include) <string>`<br>`description (regex matches, regex does not match) /regex/i`                                                                                                                                                                             | `sort by description`                       |                        |                        | `task.description`                           |
-| `has tags`<br>`no tags`<br>`tag (includes, does not include) <tag>`<br>`tags (include, do not include) <tag>`<br>`tag (regex matches, regex does not match) /regex/i`<br>`tags (regex matches, regex does not match) /regex/i`                                                                  | `sort by tag`<br>`sort by tag <tag_number>` | `group by tags`        |                        | `task.tags`                                  |
+| `has tags`<br>`no tags`<br>`tag (includes, does not include) <tag>`<br>`tags (include, do not include) <tag>`<br>`tag (regex matches, regex does not match) /regex/i`<br>`tags (regex matches, regex does not match) /regex/i`                                                                  | `sort by tag`<br>`sort by tag <tag_number>` | `group by tags`        | `hide tags`            | `task.tags`                                  |
```

Comment:
Looking at the link in the title of the Display column in Quick Reference, this would be a good place to add a reference to this new instruction:

https://publish.obsidian.md/tasks/Queries/Layout

Reply:
Updated, thanks =)